tests/socket.rs

```diff
@@ -623,7 +623,7 @@ fn recv_vectored_truncated() {
         .recv_vectored(&mut [MaybeUninitSlice::new(&mut buffer)])
         .unwrap();
     assert_eq!(received, 24);
-    assert_eq!(flags.is_truncated(), true);
+    assert!(flags.is_truncated(), "{}", true);
```

Comment:
This prints true on failure, which doesn't make a whole lot of sense.

Reply:
In fact could you revert all of these assert changes, I prefer the left/right message you get with `assert_eq`.

Reply:
Yeah, seems like we should just disable that clippy warning on this particular file.